core/filter_api.php

```diff
@@ -3166,6 +3166,25 @@ function filter_create_recently_modified( $p_days, $p_filter = null ) {
 	return filter_ensure_valid_filter( $p_filter );
 }
 
+/**
+ * Create a filter for getting issues assigned to the specified project.
+ * @param integer $p_project_id The project id or ALL_PROJECTS.
+ * @return mixed A valid filter.
+ */
+function filter_create_any( $p_project_id ) {
+	$t_filter = filter_get_default();
+
+	$t_filter[FILTER_PROPERTY_HANDLER_ID] = array( '0' => META_FILTER_ANY );
```

Comment:
Why do we need to set there 3 criteria that just say any/none?  I would expect that omitting these should deliver the same results.  @cproensa ?  Also should the approach be to create an array with desired criteria then call `filter_ensure_valid_filter()` on it?  Why trigger two calls of `filter_ensure_valid_filter()`?

Reply:
It has already been noted, that of those three, only `FILTER_PROPERTY_HIDE_STATUS => META_FILTER_NONE` is needed

>  Also should the approach be to create an array with desired criteria then call `filter_ensure_valid_filter()` on it?

Maybe filter_get_default() is not needed, and can be done with a raw array and call validate on it.
But note that this pattern is already there in other previous filter_create_xxx functions